site/src/components/Footer.tsx

```diff
@@ -42,6 +42,7 @@ const FooterNavigationLinks = FOOTER_NAVIGATION_LINKS.map(({ href, name }) => (
     component="p"
     variant="bpSmallCopy"
     sx={{
+      color: ({ palette }) => palette.gray[40],
```

Comment:
Bonus: fix link visibility in the footer

<img width="514" alt="Screenshot 2022-01-26 at 19 27 54" src="https://user-images.githubusercontent.com/608862/151232822-7bf5c0aa-272a-4a28-9123-c2fc5e26d495.png">

<img width="512" alt="Screenshot 2022-01-26 at 19 28 00" src="https://user-images.githubusercontent.com/608862/151232828-87d266e3-0e06-43ab-a39e-563dbfd95939.png">